go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/denverdino/aliyungo v0.0.0-20190125010748-a747050bb1ba
 	github.com/digitalocean/godo v1.81.0
 	github.com/dnsimple/dnsimple-go v1.0.1
-	github.com/exoscale/egoscale v1.19.0
+	github.com/exoscale/egoscale v0.97.0
```

Comment:
This looks like a downgrade, but it is actually only sticking to the latest "valid" release, as described in https://github.com/kubernetes-sigs/external-dns/issues/3460 .